package.json

```diff
@@ -15,7 +15,7 @@
     "watchfix": "tsc-watch --pretty --onSuccess \"yarn lintfix\"",
     "postinstall": "cd client && yarn",
     "i18n:build-dictionary": "tools/i18n.sh",
-    "i18n:pull": "tx pull --all --parallel --no-interactive --force",
+    "i18n:pull": "tx pull --all --parallel --no-interactive --force --mode sourceastranslation",
```

Comment:
:+1:

Reply:
Unfortunately, this doesn't actually seem to work for me.

Reply:
Your problem was due to the `minimum_perc: 95` in the `.tx/config` file